tutorials/v7/concurrentfill.cxx

```diff
@@ -25,9 +25,12 @@
 
 using namespace ROOT;
 
-double wasteCPUTime(std::mt19937 &gen)
+double wasteCPUTime()
 {
-   // Simulate number crunching through gen and ridiculous num bits
+   // Simulate number crunching through gen and ridiculous num bits in a
+   // thread-safe way
+   thread_local std::random_device rd{};
```

Comment:
Since previously the `std::mt19937` was created on the stack, it was naturally thread local.  So this change doesn't improve thread-safety and is debatable from a performance point of view

Performance:

Previously the generator was created, without (external) locking once per call to `theTask` which in this example is also once per thread and then re-use multiple time.

The new code requires,  **per iteration within `theTask`** a lookup of the thread local objects (2 of them).

Reply:
The same comment seems to apply to the next set of changes also.